src/js/components/invoice/InvoiceItemsModal.jsx

```diff
@@ -372,6 +407,14 @@ class InvoiceItemsModal extends Component {
             cache={false}
           />
         </div>
+        <div>
+          <Checkbox
```

Comment:
Perhaps a label would be handy

Reply:
Now the label is right above other checkboxes, it would look odd to add label on the left side of it.